### PR TITLE
Added slack button

### PR DESCRIPTION
**Pull request by dmjio**
cc @LukeHoersten

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -144,3 +144,7 @@ h1, h2, h3, h4, h5, h6 {
 #footerwrap a {
     color: #f2f2f2;
 }
+
+#banner {
+  margin-bottom: 40px;
+}
```
Comment (lukehoersten) [inline assets/css/style.css]: No newline?

Comment (dmjio) [reply assets/css/style.css]: Added, any more nits?

**Comment by lukehoersten** [comment]
Should we host this js file on AWS too? Why have it on heroku?

**Comment by dmjio** [comment]
It's also doing something on the server too, so unless we enabled CORS, we'll have to deliver that js from the same server. I'd rather keep all the code related to slackin in our forked repo. What I can do is transfer my forked repo to the ChicagoHaskell github group.

**Comment by dmjio** [comment]
@LukeHoersten, If you think it's too slow, I can restart the slackin server with CORS enabled, and you can host the js. What do you think about this?

**Pull request merged**